bokehjs/src/lib/models/scales/log_scale.ts

```diff
@@ -19,6 +23,20 @@ export class LogScale extends Scale {
 
   static initClass(): void {
     this.prototype.type = "LogScale"
+    this.define({
+      log_type: [ p.String, "log" ],
+    })
+  }
+
+  calculate_log(x: number): number {
+    let value: number
+    console.log(`log_type: ${this.log_type}`)
```

Comment:
This should be removed. In general, if you really need to log something, you should use bokehjs' logger.